commonItems.UnitTests/Serialization/PDXSerializerTests.cs

```diff
@@ -31,6 +31,7 @@ private class Title : IPDXSerializable {
 				{ 10, "roman_gfx" },
 				{ 5, "italian_gfx" }
 			};
+			public HashSet<string> greetings = new() { "hi", "greetings" };
```

Comment:
For the sake of having the test demonstrate it, can you also add an item that's not in the alphabetical order?

"salutations", "hi", "greetings"